### PR TITLE
Add SauceLab acknowledgement

### PR DESCRIPTION
**Pull request by nonguyen1**
Clean up readme.
Add table of contents.
Add SauceLab acknowledgement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
-To generate docs:
-<br />
+## Doc Generation
+----------------
+**To generate docs:**  
 npm run doc
+
+## Aknowledgement
+----------------
+We would like to thank [SauceLabs](https://https://saucelabs.com) for their support of open source projects like ours.
```
Comment (fmavb) [inline README.md]: There is a problem with the link: it should be https://saucelabs.com and not https://https://saucelabs.com

Comment (nonguyen1) [reply README.md]: Good Catch!!!

File: README.md [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
-#------ Doc Generation --------
-<br />
-To generate docs:
-<br />
+## Doc Generation
+----------------
+**To generate docs:**  
 npm run doc
+
+## Aknowledgement
```
Comment (rkeng) [inline README.md]: Typo: Acknowledgement

Comment (fmavb) [reply README.md]: Nice catch!

Comment (nonguyen1) [reply README.md]: Nice!

**Pull request merged**